src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          { tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={`${currentTab.id === tab.id && 'is-active'}`}
```

Comment:
let's use classNames package for such cases
<img width="350" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/70220667/4062f670-10c9-45ed-9e4f-f5f3296f8c47">